web/src/components/AssertionCard/AssertionCard.tsx

```diff
@@ -51,7 +54,16 @@ const AssertionCard: React.FC<TAssertionCardProps> = ({
           <S.SpanCountText>{spanCountText}</S.SpanCountText>
         </div>
         <S.ActionsContainer>
-          {isDraft && <S.StatusTag>draft</S.StatusTag>}
+          {isDraft && (
+            <>
+              <S.StatusTag>draft</S.StatusTag>
+              <Tooltip color="white" title="Revert Assertion">
+                <Button type="link" data-cy="assertion-action-revert" onClick={resetDefinition}>
```

Comment:
I think semantic wise it would be better to use the selector as the unique identifier.

Reply:
selector can also change, so this does not work